{"name": "Тест Авторизация с неверным email и паролем", "status": "passed", "description": "Авторизация с неверным логином или паролем выдает ошибку и код 401", "start": 1718120249804, "stop": 1718120250003, "uuid": "2247ea47-8a1b-4d33-94be-136e2cbf0672", "historyId": "04de2fb1dcbdd9c3afc97d23e09b5b03", "testCaseId": "04de2fb1dcbdd9c3afc97d23e09b5b03", "fullName": "tests.test_log_in_user.TestLoginUser#test_failed_user_login_with_invalid_credentials", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_log_in_user"}, {"name": "subSuite", "value": "TestLoginUser"}, {"name": "host", "value": "WIN-FL5IV5LS9IV"}, {"name": "thread", "value": "29088-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_log_in_user"}]}